src/PIL/Image.py

```diff
@@ -1401,19 +1401,24 @@ def getim(self):
         self.load()
         return self.im.ptr
 
-    def getpalette(self):
+    def getpalette(self, rawmode="RGB"):
         """
         Returns the image palette as a list.
 
+        :param rawmode: The mode in which to return the palette. ``None`` will
+           return the palette in its current mode.
```

Comment:
Let's add a "new in version" like in https://github.com/python-pillow/Pillow/blob/caa0deb4c76442b3f3036dbf4d59b2922deaaaab/src/PIL/ImageOps.py#L83

And should this go in release notes?